class Searchelement3
{
    public static void main(String [] args){
int arr[]={10,20,30,40,50,60};
       int length=arr.length;
       int i;
       int l=-1;
       int s=-1;
       int item1;
       
       for(i=0;i<length;i++)
       {
        if(arr[i]>l)
        {
            s=l;
            l=arr[i];
            
            
        }
        else if(arr[i]>s){
            s=arr[i];
        }

       }
       System.out.println(s);
       
        
            
        }
    
    }

    
